### PR TITLE
Package coq-inconsequential_modus_ponens.1.0.5

### PR DESCRIPTION
**Pull request by ebveeti**

### CODE REVIEW

File: released/packages/coq-inconsequential_modus_ponens/coq-inconsequential_modus_ponens.1.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+dev-repo: "git+https://github.com/ebveeti/inconsequential_modus_ponens.git"
+bug-reports: "https://github.com/ebveeti/inconsequential_modus_ponens/issues"
+maintainer: "eronenveeti174@gmail.com"
+authors: [
+  "Veeti Eronen"
+]
+license: "Creative Commons Attribution-ShareAlike 3.0 Unported"
+
+depends: [
+  "coq" {"8.15"}
```
Comment (palmskog) [inline released/packages/coq-inconsequential_modus_ponens/coq-inconsequential_modus_ponens.1.0.5/opam:17]: There needs to be a comparison operator here, such as `=` or `>=`.
```suggestion
  "coq" {>= "8.15"}
```

File: released/packages/coq-inconsequential_modus_ponens/coq-inconsequential_modus_ponens.1.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+""" # Longer description, can span several lines
+
+homepage: "https://github.com/ebveeti/inconsequential_modus_ponens"
+dev-repo: "git+https://github.com/ebveeti/inconsequential_modus_ponens.git"
+bug-reports: "https://github.com/ebveeti/inconsequential_modus_ponens/issues"
+maintainer: "eronenveeti174@gmail.com"
+authors: [
+  "Veeti Eronen"
+]
+license: "Creative Commons Attribution-ShareAlike 3.0 Unported"
```
Comment (palmskog) [inline released/packages/coq-inconsequential_modus_ponens/coq-inconsequential_modus_ponens.1.0.5/opam:14]: We try to use [SPDX](https://spdx.org/licenses/) abbreviations for licenses.
```suggestion
license: "CC-BY-SA-3.0"
```

**Review by palmskog — CHANGES_REQUESTED** [review]
The `depends` field needs to be fixed for the package to build. Also, please consider renaming the package to use `-` instead of `_`, e.g., `inconsequential-modus-ponens`.

**Comment by gares** [comment]
This development consists of 3 lemmas which are proved by tauto. We don't have a bar to pass, uit if we had one I'm sure this package would not pass it. If I understand correctly the purpose of the devolpment, I believe it would be more appropriate to turn it into a question on discourse or stackoverflow.

**Comment by ebveeti** [comment]
Hi! I'll fix the errors in the opam file a bit later, but I wanted to answer Enrico Tassi's opinion about this submission not passing the bar when it comes to what you expect from the submissions. As Enrico noticed this package only has three theorems which are all proved by tauto. Why is this the case?

I'm trying to show that modus ponens (https://en.wikipedia.org/wiki/Modus_ponens) at its heart is a tautology or with a bit weirder interpretation a conjunction. The easiest way to do this is to use the tauto. That's why the proofs are so short. I'm submitting this package because I find it strange that this logical connection hasn't been found before. The logical facts proven are very simple, but I think they are worthwhile reading. I hope that you would also accept findings this short and simple, because I think they can be meaningful too.

**Review by palmskog — APPROVED** [review]
Since CI passes and the package has reasonable metadata, I'll merge this.

**Comment by gares** [comment]
If you think the discovery is novel, you should submit it to a conference in logics, and get feedback by experts.

**Comment by ebveeti** [comment]
Thank you for your feedback. I plan on doing something like that. Since I managed to do this discovery with Coq I just wanted to put this to the opam archive first.

**Pull request merged**